databricks_cli/secrets/cli.py

```diff
@@ -0,0 +1,291 @@
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import click
+from tabulate import tabulate
+
+from databricks_cli.click_types import OutputClickType
+from databricks_cli.secrets.api import SecretApi
+from databricks_cli.utils import eat_exceptions, CONTEXT_SETTINGS, pretty_format, truncate_string, \
+    translate_value, error_and_quit
+from databricks_cli.configure.config import provide_api_client, profile_option
+from databricks_cli.version import print_version_callback, version
+
+
+SCOPE_HEADER = ('Scope', 'Backend')
+SECRET_HEADER = ('Key name', 'Last updated')
+ACL_HEADER = ('Principal', 'Permission')
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help="Creates a scope")
+@click.option('--scope', required=True)
+@click.option('--initial-manage-acl',
+              type=click.Choice(['creator-only', 'all-users']), default='creator-only',
+              help='The initial ACL applied to the scope. Must be either "creator-only"'
+                   ' or "all-users". Defaults to "creator-only".')
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_scope(api_client, scope, initial_manage_acl):
+    """
+    Creates a new scope.
```

Comment:
Create a new secret scope.

^ Add some more detail related to what an initial manage ACL means here.